runtime/Cargo.toml

```diff
@@ -6,6 +6,8 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
```

Comment:
I think we want to add a `no-cfi` feature defined here and then select `no-cfi` in dependent crates based on that. Right now, CFI is still disabled on some of the dependencies (like drivers and image-verify)

Reply:
Done